Remove init buttons

pxpm opened the pull request:
refs: #3244

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/e24f41b0-bbba-40a8-b167-00cf8f5b1d91) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/e24f41b0-bbba-40a8-b167-00cf8f5b1d91) completed: **No new issues**

tabacitu reviewed (changes requested):
I think you also included some other changes about the `select_from_array` field, by mistake. Right?

One day @pxpm , I will convince you to try to use https://git-fork.com . 
I was blind but now I see. I no longer use the command line for Git.
One day you will see the light too!!! 🤣

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/ab8eebc1-21a4-40ba-8cb8-b0dc79e4b9d6) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/ab8eebc1-21a4-40ba-8cb8-b0dc79e4b9d6) completed: **No new issues**

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/21991403-a178-449c-bdff-228d92630882) was created.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/638014bf-b0a9-4f8e-9912-7fc60b061496) was created.

pxpm commented:
@tabacitu It's open .. I just need to get used to it a little bit more hehe, thanks for the tip \o

Removed the unwanted code and added the tests for the `setRouteName()`

Best,
Pedro

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/Laravel-Backpack/CRUD/inspections/638014bf-b0a9-4f8e-9912-7fc60b061496) completed: **1 new issues, 2 updated code elements**

tabacitu reviewed (approved):
Excellent PR! Reviewed, tested, everything's AOK now.
